Simplenote/SPManagedObject+Version.swift

```diff
@@ -0,0 +1,14 @@
+import Foundation
+import Simperium
+
+// MARK: - SPManagedObject extension
+//
+extension SPManagedObject {
+
+    /// Version as an Int
+    ///
+    @objc
+    var versionInt: Int {
+        return Int(version() ?? "1") ?? 1
```

Comment:
There's a bit of duplication here. (`"1"` / `1`) How about:

```
guard let version = version() else {
    return 1
}

return Int(version)
```

(It's a bit longer, but no duplication at all!)

Reply:
Ended up like this, cause initializer of `Int` from `String` is failable (in case `String` doesn't contain a number)
```
guard let versionStr = version(), let version = Int(versionStr) else {
    return 1
}

return version
```